assets/i18n/en.json

```diff
@@ -69,5 +69,6 @@
   "Day": "Day",
   "Month": "Month",
   "Year": "year",
-  "Close": "Close"
+  "Close": "Close",
+  "SIGN_OUT_MESSAGE":"this sign out message"
```

Comment:
is this added to cy.json ? even if we do not have the translation yet, the entry should be available in both the json files. keep the text in english if translation is not available yet

Reply:
sure